cryptography/src/commonJvmAndroid/kotlin/com.wire.kalium.cryptography/CoreCryptoCentral.kt

```diff
@@ -36,12 +39,28 @@ actual suspend fun coreCryptoCentral(
         key = databaseKey
     )
     coreCrypto.setCallbacks(Callbacks())
+    setLogger(CoreCryptoLoggerImpl(), CoreCryptoLogLevel.INFO)
     return CoreCryptoCentralImpl(
         cc = coreCrypto,
         rootDir = rootDir
     )
 }
 
+private class CoreCryptoLoggerImpl : CoreCryptoLogger {
+    override fun log(level: CoreCryptoLogLevel, message: String, context: String?) {
+        when (level) {
+            CoreCryptoLogLevel.TRACE -> kaliumLogger.v("$message. $context")
```

Comment:
Is the context (json string) included here as part of the log message, or is it separate?

Reply:
the message will be concatenated - is that correct ? @johnxnguyen